CODEOWNERS

```diff
@@ -35,15 +38,18 @@ addons/packages/cert-manager @vmware-tanzu/pkg-cert-manager-owners
 ## Ownership of the contour package
 addons/packages/contour @vmware-tanzu/pkg-contour-owners
 
-## Ownership of the contour-operator package
-addons/packages/contour-operator @vmware-tanzu/pkg-contour-owners
-
 ## Ownership of the external-dns package
 addons/packages/external-dns @vmware-tanzu/pkg-external-dns-owners
 
 ## Ownership of the fluent-bit package
 addons/packages/fluent-bit @vmware-tanzu/pkg-fluent-bit-owners
 
+## Ownership of the fluxcd-kustomize-controller package
+addons/packages/fluxcd-kustomize-controller @vmware-tanzu/pkg-fluxcd-kustomize-controller-owners
+
+## Ownership of the fluxcd-source-controller package
+addons/packages/fluxcd-source-controller @vmware-tanzu/pkg-fluxcd-source-controller-owners
```

Comment:
Verified pkg-fluxcd-source-controller-owners exists, even though GitHub is complaining it doesn't.